components/dashboard/src/admin/TeamDetail.tsx

```diff
@@ -151,6 +225,89 @@ export default function TeamDetail(props: { team: Team }) {
+                onClose={() => setEditSpendingLimit(false)}
+                title="Update Usage Limit"
+                onEnter={() => false}
+                buttons={[
+                    <button
+                        className="secondary"
+                        disabled={usageLimit === costCenter?.spendingLimit}
+                        onClick={async () => {
+                            if (usageLimit !== undefined) {
+                                await getGitpodService().server.adminSetUsageLimit(
+                                    AttributionId.render(AttributionId.create(team)),
+                                    usageLimit || 0,
+                                );
+                                setUsageLimit(undefined);
+                                initialize();
+                                setEditSpendingLimit(false);
+                            }
+                        }}
+                    >
+                        Save
+                    </button>,
+                ]}
+            >
+                <p>Update the spending limit for this account.</p>
+                <div className="flex flex-col">
+                    <input
+                        type="number"
+                        min={usage?.creditsUsed}
+                        max={500000}
+                        title="Update Usage Limit"
+                        value={usageLimit}
+                        onChange={(event) => setUsageLimit(Number.parseInt(event.target.value))}
+                    />
+                </div>
+            </Modal>
+            <Modal
+                onEnter={() => false}
+                visible={editAddCreditNote}
+                onClose={() => setEditAddCreditNote(false)}
+                title="Add Credits"
+                buttons={[
+                    <button
+                        className="secondary"
+                        disabled={creditNote.credits === 0 || !creditNote.note}
+                        onClick={async () => {
+                            if (creditNote.credits !== 0 && !!creditNote.note) {
+                                await getGitpodService().server.adminAddUsageCreditNote(
+                                    AttributionId.render(AttributionId.create(team)),
+                                    creditNote.credits,
+                                    creditNote.note,
+                                );
+                                setEditAddCreditNote(false);
+                                setCreditNote({ credits: 0 });
+                                initialize();
+                            }
+                        }}
+                    >
+                        Save
+                    </button>,
+                ]}
+            >
+                <p>Adds or subtracts the amount of credits from this account.</p>
+                <div className="flex flex-col">
+                    <h4 className="mt-4">Credits</h4>
```

Comment:
**suggestion**: ~~Could we use a `label` element for all labels above inputs as we do with all other inputs?~~ Just realized there's lots of places where we need to do that, so feel free to leave this as is for now and open a follow up issue if needed.

![71lef2](https://user-images.githubusercontent.com/120486/203049436-7f5b9ac6-5ef4-4745-af97-97f629991c4c.jpg)